documentation/README.md

```diff
@@ -2,38 +2,33 @@
 
 This is **Bertly**, the DoSomething.org link shortener. We use it to create shareable URLs, like this: [`dosome.click/wq544`](https://dosome.click/wq544).
 
-Creating or revoking short-links requires an API key, which is provided as `X-BERTLY-API-Key`. If you need to integrate Bertly into a new application, ask in [#dev-bertly](https://dosomething.slack.com/messages/CAX6AJSF2/team/U025FRTM2/) or check the environment variables for the app in Lambda.
-<br>
-
+Creating or revoking short-links requires an [access token](https://github.com/DoSomething/northstar/blob/master/documentation/authentication.md) or the `X-BERTLY-API-Key` API key.
 
-Endpoint              | Functionality
---------------------- | ------------------------------------------------------------------
-`POST /`              | [Create Short-Link](#create-short-link)
-`GET /<key>`          | [Visit Short-Link](#visit-short-link) 
-`GET /<key>/clicks`   | [View Statistics](#view-statistics)
-`POST /revoke/<key>`  | [Revoke Short-Link](#revoke-short-link)
+<br>
 
+| Endpoint           | Functionality                 |
+| ------------------ | ----------------------------- |
+| `POST /`           | [Create Link](#create-link)   |
+| `GET /{link}`      | [Visit Link](#visit-link)     |
+| `GET /{link}/info` | [Inspect Link](#inspect-link) |
+| `DELETE /{link}`   | [Destroy Link](#destroy-link) |
 
-## Create Short-Link
+## Create Link
 
 ```
 POST /
 ```
 
-```sh
-# Content-Type: application/x-www-form-urlencoded
-url=https://www.github.com/dosomething/bertly
-```
+Given a URL, return a JSON object containing the shortened URL, and a token to use when revoking the shortened URL. If the same URL is provided multiple times, the same short-link will be returned. If this link is created by a non-staff user, it's restricted to a [list of safe domains](https://github.com/DoSomething/bertly/blob/master/config/domains.js).
 
-Given a URL, return a JSON object containing the shortened URL, and a token to use when revoking the shortened URL. If the same URL is provided multiple times, the same short-link will be returned.
+The request body is accepted as either `application/json` or `application/x-www-form-urlencoded`.
 
 <details>
 <summary><strong>Example Request</strong></summary>
 
 ```sh
 curl -X "POST" "https://dosome.click/" \
-     -H 'X-BERTLY-API-KEY: ******' \
-     -H 'Content-Type: application/x-www-form-urlencoded; charset=utf-8' \
```

Comment:
Does this no longer need to be specified in the request?

Reply:
Never did! It's automatically set when using the `--data-urlencode` option.

Reply:
ah I see that down there now!